### ISSUE TITLE
Comma Spacing Violation: Should not be checked in String

### ISSUE THREAD

**Issue by arsonik** [realm/SwiftLint#234]
Why would the program check if there is a space after a comma in a string ?
Example : 
```
let reg = try! NSRegularExpression(pattern: "[a-z]{2,5}", options: NSRegularExpressionOptions.CaseInsensitive)
``` 
will fail if we add a space {2, 5}

cheers & good job by the way !

**Comment by arsonik** [comment]
Also for : Opening Brace Spacing Violation: Opening braces should be preceded by a single space and on the same line as the declaration. (opening_brace)

![screen shot 2015-11-24 at 14 59 34](https://cloud.githubusercontent.com/assets/1333734/11368804/ff1af196-92bb-11e5-9558-3e7da2b312a7.png)

**Comment by bjzhou** [comment]
I have a problem like this, `let url: String = "https://www.google.com"` will fail, but `let url = "https://www.google.com"` will success

**Comment by jpsim** [comment]
@arsonik I can't reproduce your issue:

```shell
$ echo "let reg = try! NSRegularExpression(pattern: \"[a-z]{2,5}\", options: NSRegularExpressionOptions.CaseInsensitive)" | swiftlint lint --use-stdin
<nopath>:1:11: error: Force Try Violation: Force tries should be avoided. (force_try)
<nopath>:1: warning: Line Length Violation: Line should be 100 characters or less: currently 110 characters (line_length)
Done linting! Found 2 violations, 1 serious in 1 file.
$ echo "let reg = try! NSRegularExpression(pattern: \"[a-z]{2, 5}\", options: NSRegularExpressionOptions.CaseInsensitive)" | swiftlint lint --use-stdin
<nopath>:1:11: error: Force Try Violation: Force tries should be avoided. (force_try)
<nopath>:1: warning: Line Length Violation: Line should be 100 characters or less: currently 111 characters (line_length)
Done linting! Found 2 violations, 1 serious in 1 file.
```

@bjzhou I also can't reproduce your issue:

```shell
$ echo "let url: String = \"https://www.google.com\"" | swiftlint lint --use-stdin
Done linting! Found 0 violations, 0 serious in 1 file.
$ echo "let url = \"https://www.google.com\"" | swiftlint lint --use-stdin 
Done linting! Found 0 violations, 0 serious in 1 file.
```

**Comment by bjzhou** [comment]
@jpsim Don't try in command line, this issue only occurred in Xcode.

**Comment by jpsim** [comment]
SwiftLint doesn't behave differently on the command line versus Xcode.

![image](https://cloud.githubusercontent.com/assets/474794/11433037/115a62b2-946d-11e5-8b8f-00a8f3f2dd62.png)

**Comment by SokichiFujita** [comment]
@jpsim I could reproduce it. The string is for the scope string of OAuth 2.0. Please check it.

![commaviolation](https://cloud.githubusercontent.com/assets/15647722/11472148/e384caf0-97ad-11e5-9ffc-051fa1ffe40e.png)

**Comment by hankbao** [comment]
Same issue here.

**Comment by arsonik** [comment]
Looks like its happening only when checked in a large file&or project.

**Comment by jpsim** [comment]
I'm still waiting for steps to reproduce this issue, as the details shared so far are insufficient to trigger it.

**Comment by bjzhou** [comment]
```swift
//
//  Created by 周斌佳 on 15/8/1.
//  Copyright © 2015年 周斌佳. All rights reserved.
//

struct Test {
    let url = "https://www.google.com"
    let url: String = "https://www.google.com"
}
```

@jpsim  I reproduced my issue again, maybe the key is the comment. Please check it

**Comment by jpsim** [comment]
Thanks for sharing that @bjzhou, I can now reproduce the issue. I've reduced it even further:

```swift
// 周斌佳年周斌佳
let abc: String = "abc:"
```

Now I can investigate this.

**Comment by jpsim** [comment]
Should be fixed now. Thanks for reporting all! Next time, sharing a repro case really helps.
